_content/excerpts/faith-in-awakening_geoff.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Faith In Awakening"
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/books/PurityOfHeart/Section0005.html"
+drive_links:
+  - "https://drive.google.com/file/d/1BjYTgn4vfN20U5TZ49-Gf02-P-rnmJdq/view?usp=sharing"
+course: epistemology
+tags:
+  - faith
+  - psychology
```

Comment:
The psychology tag is more for the Western science. Let's go with `intellect` instead